get.scm

```diff
@@ -0,0 +1,18 @@
+(define sum-tup-in-tup
+  (lambda (tup)
+    (cond
+      ((null? tup) 0)
+      ((number? tup) tup)
+      (else (+ (sum-tup-in-tup (car tup)) (sum-tup-in-tup (cdr tup))))
+    )
+  )
+)
```

Comment:
you should not put newline between closing parenthesis because of conventions. so put all these closing parens to the end of `else` line.

the people coming from other languages (like C, JS, Java) wants to do like this.
but we don't recommend this style. there're 2 reasons.
1. we can recognize the structure by indent
2. closing parenthesis with newline style become long and we need to scroll a lot.